test/integration/tests/pauseState.ts

```diff
@@ -0,0 +1,71 @@
+import { NamedContracts } from '@custom-types/types';
+import { pauseStateConfig } from '@protocol/pauseState';
+import proposals from '@protocol/proposalsConfig';
+import { TestEndtoEndCoordinator } from '@test/integration/setup';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import { solidity } from 'ethereum-waffle';
+import { ethers } from 'hardhat';
+
+describe('e2e-pause-state', function () {
+  let contracts: NamedContracts;
+  let deployAddress: string;
+  let e2eCoord: TestEndtoEndCoordinator;
+  let doLogging: boolean;
+
+  before(async () => {
+    chai.use(CBN(ethers.BigNumber));
+    chai.use(solidity);
+  });
+
+  before(async function () {
+    // Setup test environment and get contracts
+    const version = 1;
```

Comment:
What does this version number mean?

Reply:
This has been part of our e2e harness since the beginning I think - I remember adding a version number back when I was contracting. 

Down for a refactor of our e2e setup, but that should be broken out and likely done in Foundry